### PR TITLE
Fix/node20 introduction

### PR DESCRIPTION
**Pull request by jessehouwing**
This does the Node20 taskhandler introduction without breaking and will allow a deprecation message to be added to v0 at a later point in time.

### CODE REVIEW

**Comment by jessehouwing** [comment]
Fixes #11

File: Utilites/Shell++/v1/task.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  ],
+  "execution": {
+    "Node": {
+      "target": "task.js",
+      "argumentFormat": ""
+    },
+     "Node16": {
+      "target": "task.js",
+      "argumentFormat": ""
+    }
```
Comment (madkoo) [inline Utilites/Shell++/v1/task.json:101]:
```suggestion
    },
    "Node20_1": {
      "target": "task.js",
      "argumentFormat": ""
    }
```

File: Utilites/Shell++/v1/task.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  ],
+  "demands": [
+        "sh"
+    ],
+  "version": {
+    "Major": "0",
+    "Minor": "3",
+    "Patch": "1"
+  },
+  "minimumAgentVersion": "2.144.0",
```
Comment (madkoo) [inline Utilites/Shell++/v1/task.json:21]: Should be  higher agent version then
```suggestion
  "minimumAgentVersion": "2.206.0",
```

**Comment by jessehouwing** [comment]
I upgraded to `azure-pipelines-task-lib` to also remove a bunch of security issues. Though this may require a little more stringent testing. I ran `node task.js` locally to ensure it runs, but haven't extensively tested this on the hosted agents.

**Comment by madkoo** [comment]
Why drop old node support on v1 ? The minagent version 2.206 supports node16 too

**Comment by jessehouwing** [comment]
You can add Node16 if you want to. Just make sure you drop the `Node` target, as that's for Node 6/8.
I haven't tested on Node 16.

**Comment by madkoo** [comment]
Thanks, will test this before merging

**Review by madkoo — APPROVED** [review]
Looks good.

**Pull request merged**